Projects/Features/Scene/SetCategoryScene/SetCategory/SetCategoryCore.swift

```diff
@@ -7,31 +7,75 @@
 //
 
 import Combine
+import Common
 import ComposableArchitecture
 import Foundation
+import Services
 
 public struct SetCategoryState: Equatable {
-  public init() { }
+  var allCategories: [String] = CategoryType.allCases.map {
+    $0.rawValue
+  }
+  var selectedCategories: [String]
+  var isAvailabledSelectButton: Bool {
```

Comment:
혹시
```suggestion
  var isSelectButtonEnabled: Bool {
```
라는 표현은 어떠신가요?!!

Reply:
오 네 바로 반영하겠습니다!